contracts/src/PBHEntryPoint.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {ERC1967Proxy} from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+/// @title PBH Verifier
+/// @author Worldcoin
+/// @notice An implementation of an on chain PBH verifier.
+contract PBHEntryPoint is ERC1967Proxy {
```

Comment:
Can we make it a generic proxy?

Reply:
Actually, can't we remove this contract and use `ERC1967Proxy` directly?

Reply:
Yeah that's be fine! Down for either way here. Really only helpful for differentiating namespace while testing/deploying.

Reply:
Agree with @Dzejkop, no need for additional contract